dll/cpl/input/input_list.c

```diff
@@ -662,6 +680,73 @@ InputList_Create(VOID)
     free(pLayoutList);
 }
 
+static INT InputList_Compare(INPUT_LIST_NODE *pNode1, INPUT_LIST_NODE *pNode2)
+{
+    INT nCompare = lstrcmpiW(pNode1->pszIndicator, pNode2->pszIndicator);
+    if (nCompare != 0)
+        return nCompare;
+
+    return lstrcmpiW(pNode1->pLayout->pszName, pNode2->pLayout->pszName);
+}
+
+VOID InputList_Sort(VOID)
+{
+    INPUT_LIST_NODE *pNode, *pNext, *pPrev, *pMinimum, *pList = _InputList;
```

Comment:
Splitting the long line in different "units" (makes it more readable I think?)
```suggestion
    INPUT_LIST_NODE *pList = _InputList;
    INPUT_LIST_NODE *pMinimum, *pNode;
    INPUT_LIST_NODE *pNext, *pPrev;
```

Reply:
Fixed.